contracts/prize-strategy/PeriodicPrizeStrategy.sol

```diff
@@ -479,6 +483,17 @@ abstract contract PeriodicPrizeStrategy is Initializable,
     emit RngRequestTimeoutSet(rngRequestTimeout);
   }
 
+  function setPrizePeriodSeconds(uint256 _prizePeriodSeconds) external onlyOwner requireAwardNotInProgress {
```

Comment:
Does the requireAwardNotInProgress and requiring _prizePeriodSeconds  >0 prevent all vulnerabilities?

Reply:
The prize period only affects when the prize can be awarded.  It doesn't impact the fairness or change the RNG.

Can you think of a vulnerability?